zetaclient/config/types.go

```diff
@@ -248,6 +248,9 @@ func (c Config) GetMaxBaseFee() int64 {
 func (c Config) GetMempoolCongestionThreshold() int64 {
 	c.mu.RLock()
 	defer c.mu.RUnlock()
+	if c.MempoolCongestionThreshold == 0 {
+		return DefaultMempoolCongestionThreshold
```

Comment:
In protocol operation meeting last week, I remember this feature will be disabled (in live networks) by default by setting the threshold to `0`. The value used in local e2e tests was generated by `zetaclient init` command.